docs/src/cli/compress.md

```diff
@@ -0,0 +1,72 @@
+# `compress`
+
+The `compress` command allows you to compress a WEBKNOSSOS dataset. It supports compressing specific layers and magnifications (mags) while leveraging parallel execution and customizable distribution strategies.
+
+## Usage
+
+```bash
+compress [OPTIONS] TARGET
+```
+
+### Arguments
+
+- **TARGET**  
+    Path to the WEBKNOSSOS dataset to be compressed.  
+    Example: `/path/to/dataset`.
+
+### Options
+
+- `--layer-name`  
+    Name of the layer to be compressed. If not provided, all layers will be compressed.  
+    Example: `--layer-name segmentation`.
+
+- `--mag`  
+    Magnifications (mags) to be compressed. Specify as a number or a hyphen-separated string (e.g., `2` or `2-2-2`).  
+    For multiple mags, use the option multiple times.  
+    Example: `--mag 1 --mag 2 --mag 4-4-2`.
+
+- `--jobs`  
+    Number of processes to be spawned for parallel execution.  
+    Default: Number of CPU cores.
+
+- `--distribution-strategy`  
+    Strategy to distribute the task across CPUs or nodes.  
+    Options: `multiprocessing`, `slurm`, `kubernetes`, `sequential`. 
+    Default: `multiprocessing`.
+
+- `--job-resources`  
+    Specify resources for jobs when using the SLURM distribution strategy.  
+    Example: `--job-resources='{"mem": "10M"}'`.
```

Comment:
```suggestion
    Example: `--job-resources '{"mem": "10M"}'`.
```
I think the docs should consistently use either ` ` or `=` for the key-value arguments.